MAINTAINERS.md

```diff
@@ -22,6 +22,7 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
 | Ryan Liang       | [RyanL1997](https://github.com/RyanL1997)             | Amazon      |
 | Stephen Crawford | [scrawfor99](https://github.com/scrawfor99)           | Amazon      |
 | Andriy Redko     | [reta](https://github.com/reta)                       | Aiven       |
+| Andrey Pleskach  | [willyborankin](https://github.com/willyborankin)     | Aiven       |
```

Comment:
The `./github/CODEOWNERS` also needs update, right?

Reply:
Definitely, let me fix that.